src/ElementQueries.js

```diff
@@ -12,7 +12,7 @@
     } else {
         root.ElementQueries = factory(root.ResizeSensor);
     }
-}(this, function (ResizeSensor) {
+}(typeof window !== 'undefined' ? window : this, function (ResizeSensor) {
```

Comment:
Actually now that I look at it I don't think this code should go here. The way it's written it's always favoring window if it's defined, since that typeof check happens before the and or exports checks. I'd suggest moving this to inside the else statement on line 12